src/mesh/Vertex.hpp

```diff
@@ -62,10 +68,15 @@ class Vertex {
 
   inline bool operator!=(const Vertex &rhs) const;
 
+  
+
 private:
   /// Unique (among vertices in one mesh) ID of the vertex.
   int _id;
 
+  /// Patch id that a vertex belongs to
+  int _patchid = 0;
```

Comment:
```suggestion
  int _patchID = 0;
```
would be more consistent?